packages/runtime/lib/start.js

```diff
@@ -59,7 +63,30 @@ async function startWithConfig (configManager) {
   await once(worker, 'message') // plt:init
 
   const runtimeApiClient = new RuntimeApiClient(worker)
+
+  if (config.dashboard) {
+    dashboard = await startDashboard(config.dashboard, runtimeApiClient)
+    runtimeApiClient.dashboard = dashboard
+  }
+
   return runtimeApiClient
 }
 
+async function startDashboard (dashboardConfig, runtimeApiClient) {
+  const { hostname, port, ...config } = dashboardConfig
+
+  try {
+    const dashboard = await createDashboard(config, runtimeApiClient)
+    await dashboard.listen({
+      host: hostname ?? '127.0.0.1',
+      port: port ?? 4042
+    })
+    return dashboard
+  /* c8 ignore next 4 */
+  } catch (err) {
+    console.error(err)
+    process.exit(1)
```

Comment:
I'm not convinced we should be calling `process.exit(1)` here.

Reply:
Why? If user sets a dashboard option in the config and the dashboard can't start we should terminate the process. IMHO